extensions/flyway/runtime/src/test/java/io/quarkus/flyway/runtime/FlywayCreatorTest.java

```diff
@@ -165,6 +165,18 @@ void testValidateOnMigrate() {
         assertEquals(runtimeConfig.validateOnMigrate, true);
     }
 
+    @Test
+    @DisplayName("outOfOrder is correctly set")
+    void testOutOfOrder() {
+        runtimeConfig.outOfOrder = false;
+        creator = new FlywayCreator(runtimeConfig, buildConfig);
+        assertEquals(false, createdFlywayConfig().isOutOfOrder());
```

Comment:
Not a big deal but you can use `assertFalse` here

Reply:
Yeap, let's do it!